api/projects.ts

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Po Cimrmanovsku bych řekl: Lidi chtěli TS, ale museli definovat a používat typy. 😁 

Jednoduché řešení je u otevřených objektů používat `Record<string, unknown` namísto `any`. Ideální řešení pak otypovat více detailněji.

Reply:
Hihi :) Já mám typy moc rád, ale přijde mně, že jak je ten ekosystém nemá od začátku, tak je občasné `any` zcela legitimní cesta ven. Podobně to bývalo v Objective-C s typem `id` – ve výsledku máš jakýsi „gradual typing“, ve kterém si můžeš vybrat, jak moc přesně chceš mít věci otypované. Občas není dalším striktním typováním moc co získat. `Record` jsem neznal, díky, pomohlo.